### PR TITLE
Submit action for 'No Errors Found' modal

### PR DESCRIPTION
**Pull request by piedram**
Resolves [APPEALS-64077](https://jira.devops.va.gov/browse/APPEALS-64077)

# Description
As a Caseflow User I need the ability to upload the "final PDF" to the selected Appellants eFolder endpoint in VBMS so that files can be retrieved when needed.

As a Caseflow Developer I need to;

Create a new method on submit for 'No Errors Found'
1- DocketNumber_interenalHearingID_HearingClassName.pdf
 - Found in the transcription_files table
 - aws_link column for the pdf file

2- Upload the PDF file to the Appellants eFolder location in VBMS

## Acceptance Criteria
- [x] Code compiles correctly

## Testing Plan
# Frontend

1. Open claseflow demo application.
2. Login like a admin Hearing
3. Go to demo/hearings/transcription_files
4. Switch the view and select `Hearing Admin Team cases`
5. Select any hearing with a Review Transcript task open
6. In the action Select `No errors found: Upload transcript to VBMS`
7. Add a description in the text area and click `Upload to VBMS`

### Test Coverage
Did you include any test coverage for your code? Check below:
- [x] RSpec
- [ ] Jest
- [ ] Other

### Code Climate
Your code does not add any new code climate offenses? If so why?
- [x] No new code climate issues added

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -187,6 +187,37 @@ def request_hearing_disposition_change
   end
 
   def upload_transcription_to_vbms
+    file_paths = ReviewTranscriptTask.find_by(id: task.id).appeal.hearings.first.transcription_files
+      .where(file_type: "pdf").map(&:fetch_file_from_s3!)
+
+    appeal = ReviewTranscriptTask.find_by(id: task.id).appeal
```
Comment (wmedders21) [inline app/controllers/tasks_controller.rb:193]: Again `ReviewTranscriptTask.find_by(id: task.id)` can just be `task`

Comment (piedram) [reply app/controllers/tasks_controller.rb:193]: done!

File: app/controllers/tasks_controller.rb [hunk 2]
```diff
@@ -187,6 +187,37 @@ def request_hearing_disposition_change
   end
 
   def upload_transcription_to_vbms
+    file_paths = ReviewTranscriptTask.find_by(id: task.id).appeal.hearings.first.transcription_files
```
[the remark this answers is not in the document]
Comment (piedram) [reply app/controllers/tasks_controller.rb:190]: done

**Review by mchbidwell — DISMISSED** [review]

File: app/controllers/tasks_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (mchbidwell) [reply app/controllers/tasks_controller.rb]: Creating new story to work out the details of getting the specific object

**Review by mchbidwell — COMMENTED** [review]
Please take a look at the comment 
from Will

File: app/controllers/tasks_controller.rb [hunk 3]
```diff
@@ -187,6 +187,37 @@ def request_hearing_disposition_change
...
+        complete_transcript_review
+      end
+    end
+  end
+
+  def error_found_upload_transcription_to_vbms; end
+
+  private
+
+  def create_document_params(appeal, file_name, current_file_path)
```
[the remark this answers is not in the document]
Comment (piedram) [reply app/controllers/tasks_controller.rb]: done!

**Pull request merged**